scripts/exglobal_atmos_tropcy_qc_reloc.sh

```diff
@@ -13,8 +13,8 @@
 # Make sure we are in the $DATA directory
 cd "${DATA}"
 
-tmhr=$(echo "${tmmark}"|cut -c3-4)
-cdate10=$( ${NDATE:?} -"${tmhr}" "${PDY}""${cyc}")
+tmhr=${tmmark:3:4}
```

Comment:
```suggestion
tmhr=${tmmark:2:2}
```

Because:
```bash
$ abcdefgh='abcdefgh'
$ echo $abcdefgh | cut -c3-4
cd
$ echo ${abcdefgh:3:4} 
defg
$ echo ${abcdefgh:3:2} 
de
$ echo ${abcdefgh:2:2} 
cd
```

Reply:
Thanks!